manifests/m/maforhet/ComicRackCE/nightly/maforhet.ComicRackCE.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with komac v2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: maforhet.ComicRackCE
+PackageVersion: nightly
+InstallerLocale: en-US
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{0FA63C63-846C-49B7-9A4B-553EF8EBEF0B}_is1'
+ReleaseDate: 2024-09-16
+AppsAndFeaturesEntries:
+- DisplayName: ComicRack Community Edition
+  Publisher: ComicRack Community
+  DisplayVersion: nightly-9745855
+  ProductCode: '{0FA63C63-846C-49B7-9A4B-553EF8EBEF0B}_is1'

```

Comment:
```suggestion
```
Since nightly packages update so often, it is not recommended to include AppsAndFeaturesEntries, as the automation cannot update them